data/l10n/en/main.ftl

```diff
@@ -14,6 +14,7 @@ item-is-in-inventory = You already picked up this item.
 pick-up-failed-message = You can't pick up this item.
 go-failed-message = You can't go there.
 equip-item-message = You equipped { INTER($item) }.
+unequip-item-message = You took of { INTER(§item)} and took it in your inventory.
```

Comment:
So if you wear the armour or an item in general it takes up space in your inventory, right? So you can't over-equip. Otherwise it could not work to add it to the inventory.

Somehow I don't know if I am happy about this. Imagine a pretty efficient plate-like armour. It would be heavy and take up more space in your inventory, right? If you would carry a heavy weapon too, and you can have different types of armour, the inventory has to be huge or you can't carry much else. Or, and simpler is often better, an armour could be a fixed amount of slots regardless of effectiveness.